include/script_api/common.h

```diff
@@ -18,49 +18,99 @@ extern "C" {
 /// @{
 /// @name Map
 
+/// Translates the given model's position.
 /// @evtapi
+/// @param modelID
+/// @param x
+/// @param y 
+/// @param z
 API_CALLABLE(TranslateModel);
 
+/// Rotates the model the given amount on the selected axis.
 /// @evtapi
+/// @param modelID
+/// @param angle
+/// @param x /// Act as "enablers" for if the model should be rotated on the axis or not
+/// @param y 
+/// @param z
```

Comment:
`///` is invalid as part of a param description.

```suggestion
/// @param x Whether to rotate on X axis
/// @param y Whether to rotate on Y axis
/// @param z Whether to rotate on Z axis
```